### PR TITLE
[plugin.video.s04tv] 3.1.6

### PR DESCRIPTION
**Pull request by maloep**
### Description
- update due to website changes
### Checklist:

- [X ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X ] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X ] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by Rechi** [comment]
As you marked `Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0` please also follow that rule. If you need assistance just ask.

You can ignore `Required addon plugin.video.youtube not available in current repository`, but because of that your add-on will be only seen in Helix and higher Kodi versions.

**Comment by maloep** [comment]
Sorry for that. I was quite confident that everything is correct as I just made one commit to my fork and then created the pull request. No idea why it contains more commits than this one (not that deep into git stuff as you see).

Is this the way to solve it: https://kodi.wiki/view/HOW-TO:Update_a_pull_request_to_an_official_addon_repository_on_GitHub ?

**Comment by Rechi** [comment]
The site you posted only tells how to add changes to a PR which only contains 1 commit and after that still contains only one commit.

To fix your pull request execute the following commands (origin is your repo, upstream is the xbmc repo)

```
git fetch upstream
git checkout gotham
git reset --hard upstream/gotham
git checkout origin/gotham -- plugin.video.s04tv/
git commit -m '[plugin.video.s04tv] 3.1.6'
git push -f origin gotham
```

**Comment by enen92** [comment]
Maybe at some point in time consider to bring your addon to higher repositories, gotham is already too far away and errors such as the one pointed out are likely to happen :)

File: plugin.video.s04tv/default.py [hunk 1]
```diff
@@ -132,13 +130,19 @@ def buildVideoDir(url, doc):
...
+    if not is_logged_in:
+        return
+
     isPayedContent = xbmc.getInfoLabel( "ListItem.Property(isPayedContent)" ) == 'True'
+    #payedContent requires payed Schalke TV subscription
     if(isPayedContent):
-        success = login()
-        if(not success):
+        if not has_schalke_tv:
+            xbmc.log("This video requires payed Schalke TV subscription")
```
Comment (enen92) [inline plugin.video.s04tv/default.py]: typo `paid`

**Comment by enen92** [comment]
@maloep I'd advise to submit a PR for inclusion of this addon in the Helix branch (since it depends on youtube). Another PR should also be made to mark this one (in gotham) as broken

**Comment by MartijnKaijser** [comment]
Or even to higher version as below v17 there's only a few users left.

**Pull request merged**